### ISSUE TITLE
[Accessibility] Allow users to disable Copilot inline suggestion when voice input is in progress

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#207839]
Type: <b>Feature Request</b>

CC @meganrogge 

Some users, including screen reader users, may find the Copilot inline suggestion distrupting while using voice input. It would be great if there is a custom setting for users to disable this behavior.

Extension version: 0.7.2024031110
VS Code version: Code - Insiders 1.88.0-insider (be210b3a60c7d60030c1d3d92da00d008edf6ab9, 2024-03-15T05:48:19.790Z)
OS version: Windows_NT x64 10.0.22631
Modes:

**Comment by meganrogge** [comment]
@jooyoungseo is this happening when you're using the command `Voice: Start Dictation in the Editor`? 

Can you provide the steps to reproduce?

**Comment by jooyoungseo** [comment]
In Vscode on Windows, press ctrl+alt+V to start dictation. You will see and hear the Copilot inline completion as you speak more words because Copilot suggests/predicts the next phrases.

**Comment by bpasero** [comment]
@hediet is there a setting or could we drive this with a context key? We know when voice session is in progress.

**Comment by hediet** [comment]
> is there a setting or could we drive this with a context key?

We could implement this, yes.

**Comment by meganrogge** [comment]
adding `feature-request` so it gets thoroughly verified

**Comment by meganrogge** [comment]
IMO, inline completions should be disabled when in screen reader mode.

**Comment by hediet** [comment]
> IMO, inline completions should be disabled when in screen reader mode.

Just keep in mind that we already implemented quite a few screen-reader accessibility features for inline completions!
